README.md

```diff
@@ -17,6 +17,8 @@ Read the [Nunu](https://github.com/elliejs/Willump/blob/main/tutorial/nunu.md) r
 # Willump
 Willump is a Python3 helper for the League of Legends LCU API. Willump is asynchronous and can communicate on both HTTPS and WSS channels.
 
+## for help with Willump take a look at the [Documentations folder](https://github.com/elliejs/Willump/tree/main/tutorial)
```

Comment:
Please change this from `Documentations` to `documentation`. Additionally, could you add some phrase about the contents of the folder? Really make looking at the docs appealing. If so I will add you under a contributor credit at the bottom of the readme